src/pages/myNotes/notes.jsx

```diff
@@ -1,49 +1,121 @@
+import {
+    Backdrop,
+    ChangeDescription,
+    StyledInput,
+    Container,
+    EditButton,
+    ModalWindow,
+    NotesList,
+    StyledList,
+    StyledListComponent,
+} from './styled';
 
 const Notes = () => {
+    const [todosFromLocalStorage, setTodosFromLocalStorage] = useLocalStorage('todoList', todoList);
+
+    const [todos, setTodos] = useState([]);
     const [componentInfo, setComponentInfo] = useState({});
+    const [isEditModalOpen, setIsEditModalOpen] = useState(false);
+    const [changeDescriptionInputValue, setChangeDescriptionInputValue] = useState('');
+
+    useEffect(() => {
+        setTodos(todosFromLocalStorage);
+    }, []);
+
+    const setDescription = (element) => {
+        setChangeDescriptionInputValue(element.target.value);
+    };
+
+    const submitFormChanges = () => {
+        const changedTodoList = todos.map((todo) => {
+            if (todo.id === componentInfo.id) {
+                todo.description = changeDescriptionInputValue;
+            }
+            return todo;
+        });
+
+        setTodosFromLocalStorage(changedTodoList);
+        closeModal();
+    };
+
+    const changePickedItem = (element) => {
+        getItemInfo(element);
+        openModal();
+    };
+
+    const openModal = () => {
+        setIsEditModalOpen(true);
+    };
+    const closeModal = () => {
+        setIsEditModalOpen(false);
+    };
 
     const getItemInfo = (e) => {
-        const getCurrentItem = todoList.find((todo) => {
+        const getCurrentItem = todos.find((todo) => {
             return todo.id == e.currentTarget.id;
         });
         setComponentInfo(getCurrentItem);
+        setChangeDescriptionInputValue(getCurrentItem.description);
     };
 
     return (
-        <NotesList>
-            <Container>
-                <StyledList>
-                    {todoList.map((todo) => (
-                        <StyledListComponent
-                            isActive={componentInfo && componentInfo.id === todo.id ? true : false}
-                            key={todo.id}
-                        >
-                            <ListItemText
-                                primary={
-                                    <NoteListItem
-                                        id={todo.id}
-                                        title={todo.title}
-                                        showId={false}
-                                        description={todo.description}
-                                        date={todo.date}
-                                        getItemInfo={getItemInfo}
-                                    />
-                                }
+        <>
+            <NotesList>
+                <Container>
+                    <StyledList>
+                        {todos.map((todo) => (
+                            <StyledListComponent
+                                isActive={componentInfo && componentInfo.id === todo.id}
```

Comment:
with new es syntax u don't need this componentInfo && componentInfo.id, you can replace to componentInfo?.id

Reply:
Done

Reply:
Done